### PR TITLE
Add airbnb/swift style guide

### PR DESCRIPTION
**Pull request by cruisediary**
Add airbnb/swift style guide

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ If you have used any of these styleguides, please add a comment if you can! The
 * [aturon](https://aturon.github.io)
 
 ## Swift
+* [AirBnB](https://github.com/airbnb/swift)
```
Comment (hugovk) [inline readme.md]: Minor spelling:
```suggestion
* [Airbnb](https://github.com/airbnb/swift)
```

Comment (cruisediary) [reply readme.md]: @hugovk thank you for your review
I just following `JavaScript` and `Ruby` already using `AirBnB` 🤔

Comment (hugovk) [reply readme.md]: Good point, please could you update those too?

Comment (cruisediary) [reply readme.md]: @hugovk I updated 👻

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**